frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -139,9 +142,11 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
             />
           </FormGroup>
           <AccessModeSelector
+            label={t('console-app~Access mode')}
             onChange={handleAccessMode}
             formClassName="co-clone-pvc-modal__form--space"
             pvcResource={resource}
+            volumeMode={resource?.spec?.volumeMode}
```

Comment:
Is it possible for `AccessModeSelector` to read the `volumeMode` from `resource` directly since we're already passing the resource?